lib/watcher/cpu-profile.js

```diff
@@ -5,7 +5,7 @@
 
 'use strict';
 
-var agent = require('../agent');
+var agent = require('../agent')();
```

Comment:
@sjanuary I'm sorry, I don't know how I've missed this question! I mean you need to do lazy initialization.